vignettes/rgl.Rmd

```diff
@@ -230,6 +230,37 @@ provide low-level support for `r linkfn("setAxisCallbacks")`.
 
 ## Controlling the Look of the Scene
 
+### Camera angle
+
+By default when you open a new plot:
```

Comment:
This should be "when you open a new plot with `open3d()`:".

Reply:
Done